bodhi/client/__init__.py

```diff
@@ -43,6 +43,48 @@ def _warn_if_url_and_staging_set(ctx, param, value):
     return value
 
 
+def _validate_query_user(ctx, param, value):
+    """
+    Callback used on the --user flag on query commands (query overrides and query updates)
+    to check if both --user and --mine are set, and return a warning message to stderr
+
+    Args:
+        ctx (click.core.Context): The Click context, used to find out if the --mine flag is set.
+        param (click.core.Option): The option being handled. Unused.
+        value (unicode): The value of the --user flag.
+
+    Returns:
+        unicode: The value of the --user flag.
+    """
+    if value and ctx.params.get('mine'):
+        click.echo('Warning: --mine and --user options are both set. --user will be ignored.\n',
+                   err=True)
+    return value
```

Comment:
If both validators are really needed, and since this validator only prints a warning, you can probably save writing this code twice (since the other validator has the same check) and move this warning into the bodies of the functions that use them.